app/components/telegram-auth.tsx

```diff
@@ -17,7 +17,13 @@ import { TelegramClientParams } from '@/vendor/telegram/client/telegramBaseClien
 
 const apiId = parseInt(process.env.NEXT_PUBLIC_TELEGRAM_API_ID ?? '')
 const apiHash = process.env.NEXT_PUBLIC_TELEGRAM_API_HASH ?? ''
-const defaultClientParams: TelegramClientParams = { connectionRetries: 5 }
+const appVersion = process.env.NEXT_PUBLIC_APP_VERSION ?? '1.0'
```

Comment:
rather than pull this in from an environment variable, I think we should read this directly from `package.json` - in bsky.storage we grab the version inside `next.config.ts` and stick it in the env:

<img width="348" alt="Screenshot 2025-06-26 at 8 54 50 AM" src="https://github.com/user-attachments/assets/a3d246f0-3a18-4121-aa6a-ab2efa0fd904" />

https://github.com/storacha/bluesky-backup-webapp-server/blob/main/next.config.ts#L8

which lets you use it in application code, for example:

```ts
const version = process.env.version ?? '0.0.0'
```

https://github.com/storacha/bluesky-backup-webapp-server/blob/main/src/components/services.ts#L26